ts/private/ts_project_worker.js

```diff
@@ -29,85 +29,138 @@ function getArgsFromParamFile() {
 function createFilesystemTree(root, inputs) {
     const hashes = new Map();
     const tree = {};
+    const watchingTree = {};
+
+    const TYPE = {
```

Comment:
Do these get exposed in the resulting `FilesystemTree` at all? Does it matter they are recreated for each `createFilesystemTree`?

I assume if/when this is an es6 class they will not be re-declared on every instance.

Reply:
Exactly. these would have been `const enum` if we were to use typescript. I filed #114 to come back for this.

Reply:
I assume `const enum` would be at the top of the file (outside this function). Idk if you'd want to do that here? 🤷

Reply:
Yes. it would. or a static property of the class.